dashboard/app/helpers/child_account_helper.rb

```diff
@@ -4,10 +4,10 @@ module ChildAccountHelper
   # @param request [ActionDispatch::Request] the web request
   # @return [Hash, nil]
   def parental_permission_banner_data(user, request)
-    student_lockout_date = Policies::ChildAccount.lockout_date(user, approximate: true)
+    student_lockout_date = Policies::ChildAccount.lockout_date(user, approximate: true, future: true)
 
     if request.params.key?(:show_parental_permission_banner)
-      student_lockout_date ||= Cpa::ALL_USER_LOCKOUT_DATE
+      student_lockout_date ||= Policies::ChildAccount.state_policy(user).try(:[], :lockout_date)
```

Comment:
nit: using `&.[]` is more concise than using `try`.
```suggestion
      student_lockout_date ||= Policies::ChildAccount.state_policy(user)&.[](:lockout_date)
```

Reply:
How about this?
```suggestion
      student_lockout_date ||= Policies::ChildAccount.state_policy(user)&.dig(:lockout_date)
```